.github/CONTRIBUTING.md

```diff
@@ -76,11 +76,14 @@ Check the [cakephp-codesniffer](https://github.com/cakephp/cakephp-codesniffer)
 repository to set up the CakePHP standard. The [README](https://github.com/cakephp/cakephp-codesniffer/blob/master/README.md) contains installation info
 for the sniff and phpcs.
 
-To run static analysis tools [PHPStan](https://github.com/phpstan/phpstan) and [Psalm](https://github.com/vimeo/psalm) you first have to install the additional packages via:
+To run static analysis tools [PHPStan](https://github.com/phpstan/phpstan) and [Psalm](https://github.com/vimeo/psalm) you first have to install the additional packages via [phive](https://phar.io/).
 
-    composer stan-setup
+    phive install phpstan@<version>
```

Comment:
Those commands seem rather like a regression
As a user I dont want to know the versions needed, just quickly install and run the existing versions currently locked in IMO.